Dockerfile

```
@@ -0,0 +1,13 @@
+FROM node:20
```

it's always nice to use alpine versions as they are more lightweight.

```suggestion
FROM node:apline
```